HousingRepairsOnlineApi/Controllers/RepairController.cs

```diff
@@ -29,8 +32,8 @@ public async Task<IActionResult> SaveRepair([FromBody] RepairRequest repairReque
             try
             {
                 var result = await saveRepairRequestUseCase.Execute(repairRequest);
-                appointmentConfirmationSender.Execute(repairRequest, result);
-
+                appointmentConfirmationSender.Execute(result);
+                internalEmailSender.Execute(result);
                 return Ok(result);
```

Comment:
this change will break the front end as the endpoint will return a whole saved repair object rather than just the saved repair ID.

This should fix it...
```suggestion
                return Ok(result.Id);
```